lib/Services/RoutingService.php

```diff
@@ -55,13 +55,14 @@ public static function call(string $controllerClassname, bool $authenticated = t
         if ($response instanceof Template) {
             $response->data['messages'] = $container->get(SessionMessagesService::class)->getMessages();
 
-            $response->show();
+            return $response->show();
+	    
         }
 
         if ($response instanceof ResponseInterface) {
             $emitter = new SapiEmitter();
 
-            $emitter->emit($response);
+            return $emitter->emit($response);
```

Comment:
Same

Reply:
Fixed in https://github.com/rediris-es/simplesamlphp-module-oidc/pull/36/commits/f799ec1327748d02ee8a4914e19fffdeea4f9b2c